src/app/lib/styledGrid/index.jsx

```diff
@@ -27,6 +29,136 @@ import {
   gridContainerLargeCss,
 } from '../layoutGrid';
 
+export const MediumContentContainer = styled.div`
+  @media (min-width: ${GEL_GROUP_4_SCREEN_WIDTH_MIN}) {
+    margin: 0 auto;
+    max-width: 46.4rem;
+  }
+`;
+
+export const MediumGridWrapper = ({ children }) => {
+  const columns = {
+    group0: 6,
+    group1: 6,
+    group2: 6,
+    group3: 6,
+    group4: 6,
+    group5: 12,
+  };
+
+  return (
+    <Grid enableGelGutters columns={columns}>
+      {children}
+    </Grid>
+  );
+};
+
+MediumGridWrapper.propTypes = {
+  children: node.isRequired,
+};
+
+export const GridItem = ({ children }) => {
+  const columns = {
+    group0: 6,
+    group1: 6,
+    group2: 6,
+    group3: 6,
+    group4: 6,
+    group5: 12,
+  };
+
+  const margins = {
+    group0: true,
+    group1: true,
+    group2: true,
+    group3: true,
+    group4: false,
+    group5: false,
+  };
+
+  return (
+    <Grid item columns={columns} margins={margins}>
+      {children}
+    </Grid>
+  );
+};
+
+GridItem.propTypes = {
+  children: node.isRequired,
+};
+
+export const GridItemSmall = ({ children }) => {
+  const columns = {
+    group0: 4,
+    group1: 4,
+    group2: 4,
+    group3: 4,
+    group4: 4,
+    group5: 8,
+  };
+
+  const margins = {
+    group0: true,
+    group1: true,
+    group2: true,
+    group3: false,
+    group4: false,
+    group5: false,
+  };
+
+  return (
+    <Grid item columns={columns} margins={margins}>
+      {children}
+    </Grid>
+  );
+};
+
+GridItemSmall.propTypes = {
+  children: node.isRequired,
+};
+
+export const GridItemLargeNoMargin = ({ children }) => {
+  const columns = {
+    group0: 6,
+    group1: 6,
+    group2: 6,
+    group3: 6,
+    group4: 6,
+    group5: 12,
+  };
+
+  return (
+    <Grid item columns={columns}>
+      {children}
+    </Grid>
+  );
+};
+
+GridItemLargeNoMargin.propTypes = {
+  children: node.isRequired,
+};
+
+export const GridItemMedium = ({ children }) => {
+  const columns = {
+    group0: 6,
+    group1: 6,
+    group2: 6,
+    group3: 6,
+    group4: 6,
+    group5: 10,
+  };
+
+  return (
+    <Grid item columns={columns}>
+      {children}
+    </Grid>
+  );
+};
+
+GridItemMedium.propTypes = {
+  children: node.isRequired,
+};
+
```

Comment:
I know this could be refactored and the differences be passed in as props. Its better to do that after updating everything to use `psammead-grid`